code/modules/robotics/bot/secbot.dm

```diff
@@ -1096,6 +1096,14 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
 	icon = 'icons/obj/items/items.dmi'
 	icon_state = "handcuff"
 	var/obj/machinery/bot/secbot/master
+	var/list/voice_lines = list(
+		  'sound/voice/bgod.ogg'
+		, 'sound/voice/biamthelaw.ogg'
```

Comment:
cursed commas

Reply:
extremely cursed

Reply:
But when you put them on the left it all lines up so nicely!